src/main/kotlin/com/criptext/mail/scenes/settings/profile/ProfileModel.kt

```diff
@@ -3,6 +3,6 @@ package com.criptext.mail.scenes.settings.profile
 import com.criptext.mail.scenes.SceneModel
 import com.criptext.mail.scenes.settings.profile.data.ProfileUserData
 
-class ProfileModel(var userData: ProfileUserData) : SceneModel {
-    var comesFromMailbox = false
+class ProfileModel(val comesFromMailbox: Boolean) : SceneModel {
+    lateinit var userData: ProfileUserData
```

Comment:
lateinit in a mode? I don't know rick

Reply:
I thought of using nullables, but since these specific controllers only start doing their work after they initialize their model I thought this should be no problem, and not having to check for nullity everywhere is a big plus. I can can change it though if you want.

Reply:
What about initializing it with default values? That's the normal behavior in a model

Reply:
The default value for user data needs the active account, I didn't liked the idea of accessing that though the model.

Reply:
Fill the values with empty strings, that's what I mean.